### PR TITLE
PWX-18845-pt1: Adding gp3 support

### PR DESCRIPTION
**Pull request by zoxpx**
* adding AWS/gp3 parameters

Misc:
* adding `yaml2csv.py` script that converts matrix-data to/from CSV<->YAML
* removing exec-permissions on yaml-files

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes # PWX-18845

**Special notes for your reviewer**:

According to [gp3 doc](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html), the rules are quite simple:
* gp3 delivers minimum of 3k IOPS, and maxes out at 16k IOPS at 32gb
* "flexible" IOPS between 0..32gb are calculated at 500 IOPS/gb rate

### CODE REVIEW

**Comment by zoxpx** [comment]
quick test for 7dbb781
* PX params: ` ... -s type=gp3,size=73,iops=4000 -kvdb_dev type=gp3,size=74,iops=5000`
* Amazon console volumes:
```
Name                Volume ID              Size    Volume Type      IOPS    Throughput (MB/s)  Created                          Availability Zone    State
------------------  ---------------------  ------  -------------  ------  -------------------  -------------------------------  -------------------  -------
PX...-4f23964fb1c2  vol-0edfa3f262634e9cf  73 GiB  gp3              4000                  125  May 4 2021 at 12:55:31 PM UTC-7  us-east-1c           in-use
PX...-fef199a7a916  vol-0b9f2c1505b52df7c  74 GiB  gp3              5000                  125  May 4 2021 at 12:52:22 PM UTC-7  us-east-1a           in-use
```

File: specs/decisionmatrix/yaml2csv.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import yaml
+
+
+def yaml2csv(fname):
+    ''' Converts drive-YAML syntax to CSV '''
```
Comment (trierra) [inline specs/decisionmatrix/yaml2csv.py:9]: just curious, what is the case that requires this conversion?

**Review by trierra — APPROVED** [review]
lgtm

File: specs/decisionmatrix/yaml2csv.py (hunk 1 above)
Comment (zoxpx) [reply specs/decisionmatrix/yaml2csv.py:9]: This should help with fixing/review/add the drive params in JAML-files:
* this is much easier done via Excel tables, than raw YAML
* so, one can convert params to CSV, edit in Excel/Google-Sheets, and convert back to YAML

**Review by harsh-px — APPROVED** [review]

**Comment by adityadani** [comment]
Can you add UTs for gp3 drive type here?  https://github.com/libopenstorage/cloudops/blob/master/aws/storagemanager/aws_test.go#L47

**Review by adityadani — APPROVED** [review]

**Comment by zoxpx** [comment]
Thanks all -- committing the PR to master-branch

**Pull request merged**